packages/design-system-docs/src/pages/components/Dialog/_Dialog.docs.scss

```diff
@@ -34,6 +34,27 @@ Style guide: components.dialog.react
 /*
 ---
 
+### When to use
+- Require an immediate response from the user. Use a dialog to request information that is preventing the system from continuing a user-initiated process.
+- Notify the user of urgent information. Use a modal dialog to notify the user of urgent information concerning their current work. Modal dialogs are commonly used to report system errors or convey a consequence of a user’s action.
+- Confirm a user decision. Use a modal dialog to confirm user decisions. Clearly describe the action being confirmed and explain any potential consequences that it may cause. Both the title and the button should reflect the action that will occur. If the action is destructive or irreversible then use a transactional danger modal.
+
+### When to consider alternatives
+- Every time! Before using a modal, consider whether there’s another component that might be less disruptive for the user. Modals should be a last resort.
+- Multi-step process. Avoid complicated user flows in a modal that may take the user away from the original page. A multi-step process is better suited to an individual page, guiding the user and accommodating complexities in the user flow.
+
+### Usage
+- `<Dialog>` makes use of the `<AriaModal>` component maintained by [react-aria-modal docs on Github](https://github.com/davidtheclark/react-aria-modal). The above documented props are only those directly exposed by the `<Dialog>` component, but you can pass props specific to `<AriaModal>` here as well, e.g. you can set the [`scrollDisabled`](https://github.com/davidtheclark/react-aria-modal#scrolldisabled) prop if you'd like to enable scrolling behind the modal window.
+
+### Accessibility
+
+### Focus Management
+- When the modal is opened, the entire modal is the default focus state. Most screen readers will announce the entire dialog content.
+- Focus is trapped within the modal and users can then navigate through the dialog actions with the keyboard.
+- Escape will close the modal. To disable exiting when users press the Escape key, set the `escapeExits` prop to `false`
+- When the modal closes, focus returns to the element that was focused just before the modal if activated
+- To place the focus inside of the dialog on activating the modal, set the dialog focus using the `initialFocus` prop with boolean prop `focusDialog` set to `false`
+
```

Comment:
Can we add a keyboard support section to the docs too? 
Like this
 https://github.com/CMSgov/design-system/pull/964/files#diff-5221c4608c982c50e07742b1dd1296dc451b3947e673743c31b7a2824df2182eR46

Reply:
Noticed that the keyboard navigation for modal dialog is similar to Help drawer - how does this look having keyboard navigation under accessibility section like help drawer doc?